### PR TITLE
fix(ui5-application-inquirer): Fix non-loaded i18n texts

### PR DESCRIPTION
**Pull request by IainSAP**
Fix for: https://github.com/SAP/open-ux-tools/issues/2590

- Loads bundles explicitly

### CODE REVIEW

File: packages/ui5-application-inquirer/src/i18n.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import type { TOptions } from 'i18next';
 import i18next from 'i18next';
 import translations from './translations/ui5-application-inquirer.i18n.json';
-import { addi18nResourceBundle as addInquirerCommoni18nResourceBundle } from '@sap-ux/inquirer-common';
+import { addi18nResourceBundle as addi18nResourceBundleInquirerCommon } from '@sap-ux/inquirer-common';
```
Comment (longieirl) [inline packages/ui5-application-inquirer/src/i18n.ts:4]: How is a developer going to know this?

Comment (IainSAP) [reply packages/ui5-application-inquirer/src/i18n.ts:4]: You wont, we need a TBI to fix all i18n text loading generally as this is a suboptimal solution!

Comment (longieirl) [reply packages/ui5-application-inquirer/src/i18n.ts:4]: Thanks.

**Review by longieirl — APPROVED** [review]
Changes look good.
Comments addressed.

**Pull request merged**